<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Office Equipment</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDropMode::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::DropAction::CopyAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_window"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_row"/>
    <addaction name="actionRemove_row"/>
    <addaction name="separator"/>
    <addaction name="actionCut_row"/>
    <addaction name="actionCopy_row"/>
    <addaction name="actionPaste_row"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAuthor"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionSortDeny"/>
    <addaction name="actionGrapf"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionRussian"/>
     <addaction name="actionEnglish"/>
     <addaction name="actionFrench"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="actionToolBar"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFilter"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionAdd_row">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd">
     <normaloff>:/icon/images/addrow.png</normaloff>:/icon/images/addrow.png</iconset>
   </property>
   <property name="text">
    <string>Add Row</string>
   </property>
  </action>
  <action name="actionRemove_row">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListRemove">
     <normaloff>:/icon/images/row_delete.png</normaloff>:/icon/images/row_delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailMarkImportant">
     <normaloff>:/icon/images/author.png</normaloff>:/icon/images/author.png</iconset>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew">
     <normaloff>:/icon/images/new.png</normaloff>:/icon/images/new.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen">
     <normaloff>:/icon/images/open.png</normaloff>:/icon/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowClose">
     <normaloff>:/icon/images/close.png</normaloff>:/icon/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave">
     <normaloff>:/icon/images/save.png</normaloff>:/icon/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs">
     <normaloff>:/icon/images/saveas.png</normaloff>:/icon/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentPrint">
     <normaloff>:/icon/images/print.png</normaloff>:/icon/images/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew_window">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowNew">
     <normaloff>:/icon/images/newwindow.png</normaloff>:/icon/images/newwindow.png</iconset>
   </property>
   <property name="text">
    <string>New window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/images/exit.png</normaloff>:/icon/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind">
     <normaloff>:/icon/images/find.png</normaloff>:/icon/images/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSortDeny">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRefresh">
     <normaloff>:/icon/images/sort.png</normaloff>:/icon/images/sort.png</iconset>
   </property>
   <property name="text">
    <string>Return to original sorting</string>
   </property>
  </action>
  <action name="actionGrapf">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/images/grapf.png</normaloff>:/icon/images/grapf.png</iconset>
   </property>
   <property name="text">
    <string>Grapf</string>
   </property>
  </action>
  <action name="actionCut_row">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCut">
     <normaloff>:/icon/images/cut.png</normaloff>:/icon/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Ð¡ut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy_row">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy">
     <normaloff>:/icon/images/copy.png</normaloff>:/icon/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste_row">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditPaste">
     <normaloff>:/icon/images/paste.png</normaloff>:/icon/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="Russian_language">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon/images/rus.png</normaloff>:/icon/images/rus.png</iconset>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="English_language">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon/images/eng.png</normaloff>:/icon/images/eng.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>French</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
